nms/app/packages/magmalte/app/components/admin/AuditLog.js

```diff
@@ -13,157 +13,119 @@
 
   if (error || isLoading || !response || !response.data) {
     return <LoadingFiller />;
   }
-
-  const rows = response.data.map(log => (
-    <TableRow key={log.id}>
-      <TableCell>{log.actingUserId}</TableCell>
-      <TableCell>
-        <DeviceStatusCircle
-          isGrey={false}
-          isActive={log.status === 'SUCCESS'}
-        />
-        {log.mutationType}
-      </TableCell>
-      <TableCell>{log.objectType}</TableCell>
-      <TableCell>{log.objectId}</TableCell>
-      <TableCell className={classes.actionsCell}>
-        <NestedRouteLink to={`/details/${log.id}`}>
-          <IconButton>
-            <MoreVert />
-          </IconButton>
-        </NestedRouteLink>
-      </TableCell>
-    </TableRow>
-  ));
-
   return (
-    <>
-      <div className={classes.paper}>
-        <div className={classes.header}>
-          <Text variant="h5">Audit Log</Text>
-        </div>
-        <Paper elevation={2}>
-          <Table>
-            <TableHead>
-              <TableRow>
-                <TableCell>User</TableCell>
-                <TableCell>Action</TableCell>
-                <TableCell>Object Type</TableCell>
-                <TableCell>Object ID</TableCell>
-                <TableCell className={classes.actionsColumn} />
-              </TableRow>
-            </TableHead>
-            <TableBody>{rows}</TableBody>
-          </Table>
-        </Paper>
-      </div>
-      <Route
-        path={relativePath('/details/:id')}
-        render={routeProps => (
-          <JSONDialog
-            json={
-              response.data.find(log => log.id == routeProps.match.params.id)
-                .mutationData
-            }
-            onClose={() => history.push(relativeUrl(''))}
+    <div className={classes.dashboardRoot}>
+      <Grid container spacing={4}>
+        <Grid item xs={12}>
+          <CardTitleRow
+            icon={ListAltIcon}
+            label={`Audit Logs (${response?.data?.length ?? 0})`}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <JsonDialog open={jsonDialog} onClose={onClose} row={currRow} />
+          <ActionTable
+            data={response.data}
+            columns={[
+              {title: 'User', field: 'actingUserEmail'},
+              {title: 'Action', field: 'mutationType'},
+              {title: 'Object Type', field: 'objectType'},
+              {title: 'Object ID', field: 'objectId'},
+              {
+                title: 'Status',
+                field: 'status',
+                render: currRow => (
+                  <>
+                    <DeviceStatusCircle
+                      isGrey={false}
+                      isActive={currRow.status === 'SUCCESS'}
+                    />
+                    {currRow.status}
+                  </>
+                ),
+              },
+            ]}
+            options={{
+              actionsColumnIndex: -1,
+              pageSize: 20,
+              pageSizeOptions: [20],
```

Comment:
I think we can re-use and move the constants here:
```
// default subscriber count in get subscriber query
export const DEFAULT_PAGE_SIZE = 25;

// max subscribers in array
export const MAX_PAGE_ROW_COUNT = 1000;
```

https://github.com/magma/magma/blob/master/nms/app/packages/magmalte/app/views/subscriber/SubscriberUtils.js#L39-L43

Reply:
I can move this as a constant outside this code, however this can stay local. Every component can define its own PAGE SIZE. It is more dependent on the view I would think.

Reply:
Same comment as I made below. For now, I prefer to leave it here and will add a cleanup PR where we can move all constants to a common place and import it. I think doing it piecemeal isn't useful. It is just going to move things around.